Runtime/Scripts/Cognitive3D_Manager.cs

```diff
@@ -40,7 +41,8 @@ public static Cognitive3D_Manager Instance
                     if (instance == null)
                     {
                         Util.logWarning("Cognitive Manager Instance not present in scene. Creating new gameobject");
-                        instance = new GameObject("Cognitive3D_Manager").AddComponent<Cognitive3D_Manager>();
```

Comment:
This was creating an empty game object. Is this by design? I changed it to load the actual prefab.